labonneboite/common/load_data.py

```diff
@@ -2,18 +2,12 @@
 import pickle
 import csv
 
-from labonneboite.conf import settings
-
-CACHE = {}
+from backports.functools_lru_cache import lru_cache
 
 
 def load_file(fun, filename):
```

Comment:
Je ne trouve pas le nom 'fun' très explicite... Du coup, j'ai dû regarder les fonctions appelantes pour comprendre que c'était une fonction : peut-être mettre _func_ plutôt ?

Reply:
ok will do - by the way it is legacy code :)